File: mobi_client/mobi_client/mobi_config/excel_check/temporary/java/PreDatingConfigModel.java
package com.mobi.config.configModel;
import com.mobi.config.ConfigModel;
import com.mobi.config.wrappedArrayList.*;
public class PreDatingConfigModel extends ConfigModel {
	public ArrString roleSelectSceneNames;//角色选的地点
	public ArrString roleAtLocalPointAvg;//角色在本地点avg
	public ArrString avgWithRoleSelectedScene;//角色选地点avg
	public ArrString msgWithRoleSelectedScene;//角色选地点短信内容
	public String avgWithPlayerSelectedScene;//让玩家选地点avg
	public String msgWithPlayerSelectedScene;//让玩家选地点短信内容
	public ArrString playerSelectSceneNames;//玩家选的地点
	public ArrString avgAfterPlayerSelectedScene;//对应avg
	public ArrString msgAfterPlayerSelectedScene;//对应短信内容
	public String avgWithInvitedDating;//被动约会邀请avg
	public String msgWithInvitedDating;//被动约会短信内容
}

File: mobi_client/mobi_client/Assets/scripts/Config/ConfigModel/PreDatingConfigModel.cs
namespace ConfigSystem
{
﻿    public class PreDatingConfigModel :ConfigModel {
        public System.Collections.Generic.List<string> roleSelectSceneNames;//角色选的地点
        public System.Collections.Generic.List<string> roleAtLocalPointAvg;//角色在本地点avg
        public System.Collections.Generic.List<string> avgWithRoleSelectedScene;//角色选地点avg
        public System.Collections.Generic.List<string> msgWithRoleSelectedScene;//角色选地点短信内容
        public string avgWithPlayerSelectedScene;//让玩家选地点avg
        public string msgWithPlayerSelectedScene;//让玩家选地点短信内容
        public System.Collections.Generic.List<string> playerSelectSceneNames;//玩家选的地点
        public System.Collections.Generic.List<string> avgAfterPlayerSelectedScene;//对应avg
        public System.Collections.Generic.List<string> msgAfterPlayerSelectedScene;//对应短信内容
        public string avgWithInvitedDating;//被动约会邀请avg
        public string msgWithInvitedDating;//被动约会短信内容
    }
}